article/test/ArticleFeatureTest.scala

```diff
@@ -494,7 +494,6 @@ import collection.JavaConversions._
         findFirst("aside").getAttribute("role") should be("complementary")
         findFirst("header").getAttribute("role") should be("banner")
         findFirst(".l-footer__secondary").getAttribute("role") should be("contentinfo")
-        findFirst("nav").getAttribute("role") should be("navigation")
```

Comment:
Shame that it's so broken now on IE8. It didn't used to be! I wonder how it ended up so broken within the last 6 months. Could it be the the introduction of webpack? 

Not saying we should now fix it, but it is a shame as 0.34% of the world still uses that browser. Thats a lot of people we can't reach at all now. Hopefully next year no one will be using it anymore.